ksmt-runner/src/main/kotlin/org/ksmt/solver/runner/SolverUtils.kt

```diff
@@ -0,0 +1,80 @@
+package org.ksmt.solver.runner
+
+import org.ksmt.KContext
+import org.ksmt.runner.generated.models.SolverType
+import org.ksmt.solver.KSolver
+import org.ksmt.solver.KSolverConfiguration
+import org.ksmt.solver.KSolverUniversalConfigurationBuilder
+import kotlin.reflect.KClass
+
+typealias ConfigurationBuilder<C> = (KSolverUniversalConfigurationBuilder) -> C
+
+private const val KSMT_SOLVER_PACKAGE = "org.ksmt.solver"
+
+private const val Z3_SOLVER_CLASS_NAME = "$KSMT_SOLVER_PACKAGE.z3.KZ3Solver"
```

Comment:
Wouldn't it be better to move these constants into fields of `SolverType` instances? I don't really like an idea to hardcode all these strings here since it adds additional steps (not obvious ones for me) when you attach a new solver (not a common case though)